### PR TITLE
Fix members grid CSS

### PR DESCRIPTION
**Pull request by idomand**
#93 
Hi,
I've solved the problem by changing the layout of the page from flaxbox into grid. 
I've also added a card with my details.

Unfortunately, this change had disabled the “show more” transition animation. but after talking to @MichalPorag about it she mentioned that since issue #98 is to change that animation anyway I could push this change now.

Attached below are the before and after pictures.

BeforeChange
EndResult

### CODE REVIEW

File: people/idomand.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "עידו מנדלמן",
+  "github": "idomand",
+  "facebook": "https://www.facebook.com/ido.mandelman/",
+  "linkedin": "www.linkedin.com/in/ido-mandelman",
+  "description": "מפתח פרונטאנד בהווה, עורך דין בעבר. חובב ספרי פנטזיה⚡ חתולים🐆 ובישול🥪. "
+}
```
Comment (MichalPorag) [inline people/idomand.json:7]: This is not related to this issue, please open it in new branch

**Review by MichalPorag — CHANGES_REQUESTED** [review]
Great work!

Please change the logic so the width and height of the card will be the maximum size of the cell
image

This way the gap will stay the same in all screen size.

Hint : you have gap row and column as well as gap and if you want to write it in one line you can use:
"gap: column-size row-size;".

Good luck :)

**Comment by idomand** [comment]
Hi @MichalPorag,

I have separated the extra json file into a different branch and fixed the grid-gap and the height and width of the cards as you requested.

File: styles/Home.module.scss [hunk 2]
```diff
@@ -7,21 +7,15 @@
...
     justify-items: center;
-    transition: 1s ease;
-    column-gap: 30px;
-    & > [class^="person"] {
-      margin-bottom: 45px;
-    }
   }
 }
 
 @media only screen and (max-width: $smTablet) {
```
Comment (MichalPorag) [inline styles/Home.module.scss:15]: You can remove all the media query from here

File: components/personCard/personCard.module.scss [hunk 3]
```diff
@@ -1,9 +1,10 @@
-@import '../../styles/media';
+@import "../../styles/media";
 
 .card {
-  width: 550px;
-  height: auto;
+  min-width: 550px;
```
Comment (MichalPorag) [inline components/personCard/personCard.module.scss:4]: You can remove this

File: styles/Home.module.scss [hunk 4]
```diff
@@ -4,23 +4,18 @@
...
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
-
-    & > [class^="person"] {
-      margin-bottom: 45px;
-      margin-left: 30px;
-    }
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(550px, 1fr));
```
Comment (MichalPorag) [inline styles/Home.module.scss:8]: Please change to minmax(350px, 1fr)

**Review by MichalPorag — CHANGES_REQUESTED** [review]
@idomand 
Great job!
We are getting closer, just a little bit improvements :) 

After you change that things, please check out the card height issue and the gap issue.

Hint: you need to remove the duplication gap declaration.

**Review by MichalPorag — APPROVED** [review]

**Pull request merged**